subdomains/blog/_posts/2020-03-30-voltas-helpful-error-messages.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Volta's Helpful Error Messages"
+date:   2020-03-30 08:00:00 -0700
+categories: [errors, delight]
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+Nothing is quite so frustrating as a piece of technology giving you an error message that you don’t understand. We’ve all felt that surge of anger like we were a character in _Office Space_, “’PC Load Letter,’ what the $&@# does that mean?!” With Volta, we’re committed to having helpful errors. In fact, **if you ever find yourself frustrated or confused by an error message from Volta, we consider that a bug**, so please file an issue!
+<!--more-->
+
+Early on in the development of Volta, we revamped our error-handling code so that every error in the program is associated with a custom error message. These error messages provide context of what Volta was trying to do when the error occurred and they all provide a call-to-action for what you can do to solve the problem. For example, if you try to run `node` before installing a version of Node, you don’t see a generic `command not found` message, you see this:
+
+<section style="text-align: center;">
+  <img src="/assets/volta_helpful_error.png" alt="Error message when trying to run node without an installed version." /><br>
```

Comment:
For the sake of readers who can't see, can we provide an alternative here?

- perhaps in-flow and so visible to screen readers but invisible to normal users
- perhaps just a text snippet instead of a screenshot, therefore equally accessible to everyone?

Reply:
Updated to use the markup from a code block, so it's all text instead of a screenshot.

Reply:
:clap: Will review the rest tomorrow!